LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,256 +1,106 @@
-# Slint Software License Agreement
+# Slint Software License, version 2.0
 
-Version 2.0
+## Preamble
 
-Copyright © 2023 SixtyFPS GmbH <https://slint.dev/imprint.html>
+Slint is a toolkit that can be used to build user interfaces for applications.
+Slint (hereafter referred to as "Software") is made available under
+different licenses by SixtyFPS GmbH incorporated at Oranienburger Str. 44, 16540
+Hohen Neuendorf, Germany ("SixtyFPS", "us", "we" or "our").
 
-SixtyFPS GmbH with offices at Oranienburger Str. 44, 16540 Hohen Neuendorf,
-Germany (**SixtyFPS**, **us**, **we** or **our**) grants a license to the
-software (**Licensed Software**), as specified herein, subject to the Terms and
-Conditions of this Slint Software License Agreement (**Agreement**).
+The "Slint Software License" is suitable for those who develop applications and
+do not want to use Slint open source components under copyleft licenses.
 
-By using the Licensed Software, **you** agree to be bound by this Agreement.
-**If you do not agree, do not use the Licensed Software.** If you are accessing
-and using the Licensed Software on behalf of a company (such as your employer)
-or other legal entity, you represent and warrant that you have the authority to
-bind that company or other legal entity to this Agreement. In that case, **you**
-and **your** will refer to that company or other legal entity.
+By using the Software, You agree to be bound by this license.
```

Comment:
I'm curious how this phrasing works in conjunction with multiple licenses. If the mere use of the software implies that I'm bound by this commercial license, then how do I use Slint _not_ under this license but say the GPL?

Is this somehow related to distribution or the presence of this license in the distribution?

Reply:
Removed sentence.